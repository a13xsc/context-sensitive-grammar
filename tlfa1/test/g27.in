         (     {     E    ,       B      ,   %         ,'  ,      I        ,   9     ,   C      ,          P    ,         Y        ,     Q ,          Z       ,         U     ,      c          ,          A    ,  N   ,   M,         G     ,     D,   J     ,    X                }      ,      {        c    ,  9     ,         %          ,         '                }             ,          {          (         D ,    BJ'AZEE%GBAPQ )     ,          (    M     ,        GMDIP   )      ,     (          M      ,     e     ),  (       M   ,        JM'9cY'McMMZUZU )      , (        Y      ,          P9N   )                   }              ,      M      )      
