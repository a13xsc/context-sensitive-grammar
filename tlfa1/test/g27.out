G
A
D
U
C
Q
X
I
Z
B
E
Y
J
N
P
